src/main/java/io/nats/client/impl/StringAuthHandler.java

```diff
@@ -14,11 +14,13 @@
 package io.nats.client.impl;
 
 import io.nats.client.AuthHandler;
-import io.nats.client.NKey;
+import io.nats.nkey.NKey;
```

Comment:
Safe to use change since NKey is internal to class and not returned in any public api of the class.